var/spack/repos/builtin/packages/heppdt/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class Heppdt(Package):
+    """ The HepPID library contains translation methods for particle ID's
+        to and from various Monte Carlo generators and the PDG standard
+        numbering scheme. We realize that the generators adhere closely
+        to the standard, but there are occasional differences."""
```

Comment:
I think this is the last thing that the documentation tests are unhappy about. When building the documentation, we run `spack package-list` to create [this beautiful page](http://spack.readthedocs.io/en/latest/package_list.html). `spack package-list` prints out an .rst file based on the docstrings of every package. But if the docstring is improperly formatted, the indentation in `package_list.rst` can be messed up. To fix this one, I think you need to remove the leading spaces so it looks like:
```python
"""The HepPID library contains translation methods for particle ID's
to and from various Monte Carlo generators and the PDG standard
numbering scheme. We realize that the generators adhere closely
to the standard, but there are occasional differences."""
```
By the way, if you want to run these tests locally, there's a `share/spack/qa/run-doc-tests` command that will try building the documentation and report any errors. You just need sphinx, graphviz, git, mercurial, and svn installed. All of them should be in Spack.